### PR TITLE
firms nrt

### PR DESCRIPTION
**Pull request by norlandrhagen**

### CODE REVIEW

**Comment by norlandrhagen** [comment]
@maxrjones & @katamartin the CI is now writing 9 pyramid levels to `s3://carbonplan-forest-offsets/web/tiles/current-firms-hotspots`.  How do y'all feel about merging?

**Comment by katamartin** [comment]
@norlandrhagen no concerns on my end!

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ name: Build and push container image
 
 on:
   pull_request:
-    paths:
-      - 'binder/**.yml'
+  paths:
+    - 'binder/**.yml'
```
Comment (maxrjones) [inline .github/workflows/build.yaml:6]:
```suggestion
    paths:
      - 'binder/**.yml'
```
I think the indentation is required here (https://github.com/carbonplan/forest-offsets-fires/actions/workflows/build.yaml)

File: .github/workflows/firms-nrt.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+name: FIRMS NRT
+
+on:
+  pull_request:
```
Comment (maxrjones) [inline .github/workflows/firms-nrt.yaml:4]:
```suggestion
  # pull_request:
```

**Review by maxrjones — APPROVED** [review]
This is great, thanks @norlandrhagen! I think it would be best to remove the unused firms-thermal-anomalies.ipynb notebook and firms_nrt_ETL.py before merging.

**Comment by norlandrhagen** [comment]
Thanks for the feedback @maxrjones!

**Pull request merged**